lib/js_locale_helper.rb

```diff
@@ -163,7 +163,17 @@ def self.output_MF(locale)
       require("discourse-mf");
     JS
   rescue => e
-    Rails.logger.error("Failed to compile message formats for #{locale} '#{e}'")
+    message_formats[locale.to_s]
+      .filter_map do |key, value|
+        next if MessageFormat.compile(locale, value, strict: false)
+      rescue StandardError
+        key
+      end
+      .then do |strings|
+        Rails.logger.error(
+          "Failed to compile message formats for #{locale} '#{e}'.\n\nBroken strings are: #{strings.join(", ")}",
```

Comment:
Should we also add the original error message here? It might be useful for debugging.

Something like:
```suggestion
          "Failed to compile message formats for #{locale} '#{e}'.\n\nBroken strings are: #{strings.join(", ")}\n\nError: #{e}",
```

Reply:
It’s already there :sweat_smile: (`'#{e}'`)
It might be better to present it like you suggest, though :thinking:

Reply:
Oh.... right... dunno how I missed that 🤦 

I don't have any string feeling on the formatting, as long as it's somewhere 👌